components/subchartsdemo/README.md

```diff
@@ -0,0 +1,255 @@
+# Example: subcharts
+
+This folder contains an example how to deploy an application consistsing of multiple helm charts. It consists of a helm chart, a [toi package](../../docs/reference/ocm_toi-bootstrapping.md) and a reference to the [helm helminstaller](../helminstaller/README.md).
+
+The helm chart has multiple other charts as dependencies and has no templates. Its only purpose is to act as an anchor for deployment. The toi package is used to install the component with the helm-installer using the `ocm bootstrap` command.
+
+## Helm Charts with dependencies
+Goal of this deployment is to install two different components having their own helm-charts: echoserver and pod-info. Both have their own component version and a helm-chart resource. The chart dependencies are declared in the `packagespec.yaml` as `subcharts`:
+
+```yaml
+executors:
+  - ...
+    config:
+      ...
```

Comment:
I think a root chart is required here, also. ???

Reply:
full config added